packages/generators/init-generator.ts

```diff
@@ -81,15 +86,22 @@ export default class InitGenerator extends Generator {
 			"\n"
 		);
 
+		const initConfig = []
+
 		return this.prompt([Confirm("entryType", "Will your application have multiple bundles?", false)])
 			.then(
 				(entryTypeAnswer: { entryType: boolean }): Promise<{}> => {
+					this.configuration.config.initConfig.push(entryTypeAnswer);
+					initConfig.push(entryTypeAnswer);
```

Comment:
Should we remove the `initConfig.push` if we are saving the answers to `this.configuration.config.initConfig`? :)

Reply:
Correct! Thanks, just pushed. :)

Yeah `this.configuration.answers` sounded better/ 👍